tasks.py.jinja

```diff
@@ -309,6 +354,115 @@ def start(c, detach=True, debugpy=False):
     _logger.info("Waiting for services to spin up...")
     time.sleep(SERVICES_WAIT_TIME)
 
+@task(
+    develop,
+    help={
+        "modules": "Comma-separated list of modules to test. Default: 'base'.",
+        "core": "Install all core addons. Default: False",
+        "extra": "Install all extra addons. Default: False",
+        "private": "Install all private addons. Default: False",
+    },
+)
+def install(c, modules=None, core=False, extra=False, private=False):
+    """Install Odoo addons"""
+    if not (modules or core or extra or private):
+        raise exceptions.ParseError(
+            message="You must provide at least one option. "
+            "See --help for details."
+        )
+    cmd = "docker-compose run --rm addons init"
+    if core:
+        cmd += " --core"
+    if extra:
+        cmd += " --extra"
+    if private:
+        cmd += " --private"
+    if modules:
+        cmd += f" -w {modules}"
+    with c.cd(str(PROJECT_ROOT)):
+        c.run(
+            cmd,
+            env=dict(
+                UID_ENV,
+            ),
+        )
+
+
+def _get_cwd_addon(file=None):
+    if not file:
+        cwd = Path.cwd()
+    else:
+        cwd = Path.parent(file)
+    manifest_file = False
+    while not manifest_file:
```

Comment:
This check is not used, because if a `manifest_file` is found, the `return` exits the `while`.

Instead:

```suggestion
    while PROJECT_ROOT < cwd:
```

And remove the same test you do at the end of the loop.